pkg/authenticator/authenticator.go

```diff
@@ -9,6 +9,7 @@ import (
 	"encoding/asn1"
 	"encoding/pem"
 	"fmt"
+	"github.com/cyberark/conjur-authn-k8s-client/pkg/access_token"
```

Comment:
Some linting comments:

1. Splitting imports to "go standard", "external deps" and "internal" improves readability of dependencies
```
import (
  ## go standard
  "encoding/pem"

  # external
  "gopkg.in/yaml.v2"

  # internal
  "github.com/cyberark/conjur-authn-k8s-client/pkg/access_token"
)
```

2. Similar to `#1`, sort (alphabetically) and categorise aggressively (imports, struct field names etc.). Below the struct definition is split into private fields and public fields, sorted alphabetically.
```
type Authenticator struct {
  client             *http.Client
  privateKey         *rsa.PrivateKey
  AccessTokenHandler access_token.AccessTokenHandler
  Config             authnConfig.Config
  PublicCert         *x509.Certificate
}
```

Reply:
done